gateway/webhook/webhook.go

```diff
@@ -261,6 +263,36 @@ func (webhook *HandleT) batchRequests(sourceDef string, requestQ chan *webhookT)
 	}
 }
 
+func prepareRequestBody(req *http.Request, includeQueryParams bool, sourceType string, sourceListForParsingParams []string) ([]byte, error) {
+	body, err := io.ReadAll(req.Body)
```

Comment:
Didn't see where are we closing the body?

Reply:
addressed